package.json

```diff
@@ -8,10 +8,9 @@
   "type": "module",
   "scripts": {
     "start": "vite serve",
-    "build": "vite build",
-    "pack": "rm -rf dist && npm run build && npm run post:pack",
-    "local:pack": "npm run pack && cd dist && npm pack && mv *.tgz ../",
-    "post:pack": "node ./scripts/post-pack.js",
+    "build": "vite build --mode production && npm run post:build",
+    "post:build": "node ./scripts/post-pack.js",
```

Comment:
just call this `postbuild` (no colon) and then you don't have to call `npm run post:build` - it becomes automatic:

https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts

Reply:
@vespasianvs changed